pages/services/spark/2.5.0-2.2.1/release-notes/index.md

```diff
@@ -0,0 +1,202 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes
+title: Release Notes
+menuWeight: 145
+excerpt: Release notes for DC/OS Apache Spark 2.5.0-2.2.1
+featureMaturity:
+---
+
+## Version Spark and Spark History 2.5.0-2.2.1
+
+### New Features
+- Unique Mesos Task IDs for Spark executors
+- Trusted Ubuntu 18.04 base Docker image
+- `nobody` user support on RHEL/CentOS (via configuration)
```

Comment:
Compare previous release notes to this version. These should be worded similarly eg 
- `Added unique Mesos Task IDs for Spark executors`
- `Added trusted Ubuntu 18.04 base Docker image`... etc

Same for updates:
- `Updated JRE to 1.8.192`
- `Upgraded Ubuntu to 18.04`

Reply:
Done